src/Form/CommitteeFeedMessageType.php

```diff
@@ -15,14 +15,16 @@ class CommitteeFeedMessageType extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder
-            ->add('subject', TextType::class, [
+        if (CommitteeMessage::class === $options['data_class']) {
```

Comment:
There should be a separate form that inherits from this one and just add the new field (or remove the extra field if it's the exception).